lib/types.nix

```diff
@@ -14,7 +14,10 @@ let
       name = "strLua";
       inherit description;
       descriptionClass = "noun";
-      check = v: lib.isString v || isRawType v;
+      check = lib.any [
```

Comment:
still need the v: references in these lib.any changes?

Reply:
Yeah, I forgot what [`builtins.any`](https://noogle.dev/f/builtins/any) actually does.

To do this I'd need `x: any (pred: pred x) [ /* predicates */ ]`, or fallback to plain old `x: pred1 x || pred2 x || pred3 x`